src/client/components/Footer/footer.css

```diff
@@ -0,0 +1,48 @@
+footer {
+  width: 100%;
+  background-color: #929B90;
+}
+
+footer .footer {
+  display: flex;
+  justify-content: right;
+  align-items: center;
+  padding-right: 10%;
+}
+
+footer .footer .column {
+  padding: 4%;
+}
+
+footer h3 {
+  color: white;
+}
+
+footer ul {
+  list-style-type: none;
+  display: inline;
+}
+
+footer li {
+  line-height: 28px;
+  color: white;
+  text-decoration: none;
+  transition: 0.75s;
+  font-family: inter;
```

Comment:
This font family is intended to be used in the whole project, so it will be better to set it so everyone can use it, then you will not need to specify it here.